frontend/src/types/analytics.d.ts

```diff
@@ -136,7 +136,12 @@ declare global {
 
   interface IEventAndCondition {
     readonly eventName: string;
-    readonly conditions?: Condition[];
+    readonly conditions?: ICondition[];
+    readonly conditionOperator?: 'and' | 'or';
```

Comment:
should be `conditionOperator: 'and' | 'or'`?

Reply:
done.